content/en/docs/concepts/sdk-configuration/sdk-environment-variables.md

```diff
@@ -0,0 +1,142 @@
+# OpenTelemetry Environment Variable Specification
+
+**Status**: [Mixed](document-status.md)
+
+The goal of this specification is to unify the environment variable names between different OpenTelemetry SDK implementations. SDKs MAY choose to allow configuration via the environment variables in this specification, but are not required to. If they do, they SHOULD use the names listed in this document.
+
+## Parsing empty value
+
+**Status**: [Stable](document-status.md)
+
+The SDK MUST interpret an empty value of an environment variable the same way as when the variable is unset.
+
+## Special configuration types
+
+**Status**: [Stable](document-status.md)
+
+### Boolean value
+
+Any value that represents a Boolean MUST be set to true only by the case-insensitive string `"true"`, meaning `"True"` or `"TRUE"` are also accepted, as true.
+An SDK MUST NOT extend this definition and define additional values that are interpreted as true.
+Any value not explicitly defined here as a true value, including unset and empty values, MUST be interpreted as false.
+If any value other than a true value, case-insensitive string `"false"`, empty, or unset is used, a warning SHOULD be logged to inform users about the fallback to false being applied.
```

Comment:
Same comment as above, wondering why use 'should' here vs 'must'. If the content here is trying to inform best practices, it's clearer to outline them as 'must' (especially so in the case of error messages)

Reply:
I'll change all instances for 'should' to 'must' as we are trying to enforce the best practices for naming convention